Mathlib/RingTheory/Polynomial/Pochhammer.lean

```diff
@@ -142,14 +142,6 @@ theorem ascPochhammer_succ_comp_X_add_one (n : ℕ) :
 set_option linter.uppercaseLean3 false in
 #align pochhammer_succ_comp_X_add_one ascPochhammer_succ_comp_X_add_one
 
-
--- TODO: find a better place for this lemma?
-theorem Polynomial.mul_X_add_nat_cast_comp {p q : S[X]} {n : ℕ} :
```

Comment:
There was an analogous todo for subtraction, right? Can you please move that one as well in this PR?

Reply:
Yes, done.